cmd/eksctl/create.go

```diff
@@ -66,6 +66,7 @@ func createClusterCmd() *cobra.Command {
 
 	fs.StringVarP(&cfg.Region, "region", "r", api.DEFAULT_EKS_REGION, "AWS region")
 	fs.StringVarP(&cfg.Profile, "profile", "p", "", "AWS credentials profile to use (overrides the AWS_PROFILE environment variable)")
+	fs.IntVarP(&cfg.NodeVolumeSize, "node-volume-size", "", 0, "Node volume size (in GB)")
```

Comment:
It would be good to use a constant, e.g. `api.DEFAULT_NODE_VOLUME_SIZE`.

Reply:
Also, please move it just above `cfg.MaxPodsPerNode`, to match the order in which it's defined in the struct.

Reply:
The default behavior here is to use the volume size from the AMI. I don't know if we need a named constant for the empty value.